cpu/native/irq_cpu.c

```diff
@@ -229,7 +229,7 @@ int irq_is_in(void)
 int _native_popsig(void)
 {
     int nread, nleft, i;
-    int sig;
+    int sig = -1;
```

Comment:
"0" would be a better choice, as the only user (right below) uses this result as index into native_irq_handlers, which has a safe NULL as default in native_irq_handlers[0], but native_irq_handlers[-1] would segfault.

Reply:
ok!